test/system/test_tpm2_getmanufec.sh

```diff
@@ -76,6 +76,7 @@ tpm2_getmanufec -H $handle -U -E ECcert2.bin -f test_ek.pub -o $opass -e $epass
                 https://ekop.intel.com/ekcertservice/
 
 tpm2_listpersistent | grep -q $handle
+tpm2_listpersistent
```

Comment:
Why you are adding this to the test? At least it should be mentioned in the commit message.

If is just to see if the friendly attribute names are printed, I don't think is the correct way to do it. Instead we need a tpm2_listpersistent integration test as mentioned in issue #489.

Reply:
This is a change I had locally that I never meant to submit just for getting tpm2_listpersistent output with an object defined.